pkg/apis/v1beta1/nodepool_validation_cel_test.go

```diff
@@ -57,6 +53,9 @@ var _ = Describe("CEL/Validation", func() {
 			},
 		}
 	})
+	BeforeEach(func() {
```

Comment:
```suggestion
	AfterEach(func() {
```

This is `AfterEach` in all the other `suite_test.go`s and this top-level cleanup should generally sit in the `suite_test.go` file